pdseg/models/model_builder.py

```diff
@@ -175,12 +175,16 @@ def build_model(main_prog, start_prog, phase=ModelPhase.TRAIN):
             # 在导出模型的时候，增加图像标准化预处理,减小预测部署时图像的处理流程
             # 预测部署时只须对输入图像增加batch_size维度即可
             if ModelPhase.is_predict(phase):
-                origin_image = fluid.data(
-                    name='image',
-                    shape=[-1, -1, -1, cfg.DATASET.DATA_DIM],
-                    dtype='float32')
-                image, valid_shape, origin_shape = export_preprocess(
-                    origin_image)
+                if cfg.SLIM.PREPROCESS:
+                    image = fluid.layers.data(
```

Comment:
Migrate `fluid.layers.data` to `fluid.data`

Reply:
done